nncf/torch/layer_utils.py

```diff
@@ -19,6 +24,33 @@
 COMPRESSION_MODULES = Registry("compression modules")
 
 
+class StatefullTorchModuleInterface(ABC):
```

Comment:
IMHO: This is a non-intuitive class name. Please think about more suitable name.

May be:
`IModuleConfig`, `ModuleConfigInterface`, `StatefullModuleInterface`...

Reply:
I picked `StatefullModuleInterface`, renamed